doc/faq.rst

```diff
@@ -136,10 +136,10 @@ If a peak is not detected, in the power spectrum, within a given frequency band,
 consistent with there being no periodic activity at that frequency. Without a detected peak,
 we argue that there is no evidence of periodic activity, at that frequency, over and
 above the power as expected by the aperiodic activity. In this situation, one should be very
-wary of interpreting activity at this frequency, as it is most likely to reflect aperiodic
+wary of interpreting activity at this frequency, as it is most likely reflects aperiodic
 activity.
 
-However, we can, of course not prove a negative, and the absence of a detected peak does
+However, we cannot prove a negative, and the absence of a detected peak does
```

Comment:
I think there is still a phrasing issue here. Right now, leading on to the next line, it says:
```
However, we cannot prove a negative, and the absence of a detected peak does therefore imply that there must be no periodic activity at that frequency band.
```
Where there should be a 'not' in there. 
Can you updated it to read as:
```
However, we cannot prove a negative, and so the absence of a detected peak does not imply that there is definitively no periodic activity at that particular frequency.
```
^I also did some work edits, feel free to edit a bit more, but the jist of it should be that we can not prove there is no periodic activity (just that we want to recommend the 'preferred' interpretation in this case would be that it is aperiodic.